screens/VehicleDetailsScreen.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useState } from "react";
+import { ActivityIndicator, StyleSheet } from "react-native";
+import { Text, View } from "../components/Themed";
+import { Vehicle } from "../models";
+import fetchAPI from "../fetchApi";
+
+interface DetailsItemProps {
+  name: string;
+  value: string;
+}
+
+const DetailsItem = ({ name, value }: DetailsItemProps) => (
+  <Text style={styles.listItem}>
+    {name}: {value}
+  </Text>
+);
+
+const emptyVehicle = (): Vehicle => ({
+  name: "---",
+  model: "---",
+  manufacturer: "",
+  cost_in_credits: "",
+  max_atmosphering_speed: "",
+  passengers: "",
+});
+
+export default function VehicleDetailsScreen({ navigation, route }) {
+  const [isLoading, setLoading] = useState(true);
+  const [data, setData] = useState<Vehicle>(emptyVehicle);
```

Comment:
emptyVehidle jest tutaj nie potrzebne (może nawet generawać pewne problemy - przypomnij mi to tym kiedyś, bo to dłuższa opowieść ;) )
użyj: `useState<Vehicle | undefined>(emptyVehicle);`